### PR TITLE
Fix Readme

### PR DESCRIPTION
**Pull request by bbiiggppiigg**
1. Separated installation commands that need sudo privilege from those need not.
2. Add git as dependency.
3. Add fix to bower installation problem with git protocol => use https protocol.

### CODE REVIEW

**Comment by nickelcheng** [comment]
That's good.
But you need to merge the latest code to your repo so that git can auto merge your PR.

File: README.md [hunk 1]
```diff
@@ -30,32 +35,37 @@ git clone git@github.com:bruce3557/NTHUOJ_web.git
...
+sudo apt-get install mysql-server
+sudo apt-get install python-pip
+sudo apt-get install python-mysqldb
+sudo apt-get install nodejs
+sudo apt-get install npm
+sudo npm install -g bower
+sudo ln -s /usr/bin/nodejs /usr/bin/node
+sudo pip install django==1.7
+sudo pip install django-bower
+sudo pip install django-axes
```
Comment (henryyang42) [inline README.md]: For our current deployment, we use a [virtualenv](https://virtualenv.pypa.io/en/latest/) to create isolated Python environments. We can also put all Python dependencies in a ``requirements.txt`` and then use ``pip install -r requirements.txt`` without sudo. Should we also change fix this?

Comment (nickelcheng) [inline README.md]: Both is ok for me.
Use ```requirements.txt``` in this PR or in another PR.
It's up to @bbiiggppiigg and @henryyang42

Comment (henryyang42) [inline README.md]: I'll fix it after it's merged.

File: README.md [hunk 2]
```diff
@@ -28,42 +33,48 @@ git clone git@github.com:bruce3557/NTHUOJ_web.git
...
+sudo pip install django-autocomplete-light
+sudo pip install django-datetime-widget
+sudo pip install django-bootstrap
+sudo pip install django-bootstrap-form
+sudo pip install django-ckeditor
+sudo pip install requests
+sudo pip install -I pillow
+sudo apt-get install git
+=======
```
Comment (nickelcheng) [inline README.md]: Please remove this line.

**Comment by nickelcheng** [comment]
+1, going to merge.

**Pull request merged**